src/components/container/BloodPressureVisualization/BloodPressureVisualization.tsx

```diff
@@ -280,22 +286,17 @@ export default function (props: BloodPressureVisualizationProps) {
         return Category.Unknown;
     }
 
-    function pageWeek(newStart: Date) {
-        setStartOfWeek(newStart);
-    }
-
-    useInitializeView(initialize, [], [props.previewState]);
+    useInitializeView(initialize, [], [props.previewState, props.weekStartsOn, dateRangeContext?.intervalType]);
```

Comment:
Is `dateRangeContext?.intervalType` necessary here?

I don't understand why we would need to watch for it changing in order to reload the data when only a single `intervalType` value would ever allow us to get to this point in the code.